components/brave_new_tab_ui/containers/newTab/settings/sponsoredImagesToggle.tsx

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import styled from 'styled-components'
+
+import '$web-components/app.global.scss'
+
+// Common components
+import { SettingsText } from '../../../components/default'
+import { Toggle } from '../../../components/toggle'
+
+// Assets
+import InfoIcon from '../../../components/default/settings/assets/info-icon.svg'
+
+// Utilities
+import { getLocale } from '../../../../common/locale'
+
+interface Props {
+  onChange: () => void
+  onEnableRewards: () => void
+
+  checked: boolean
+  disabled: boolean
+  rewardsEnabled: boolean
+  adsEnabled: boolean
+  canSupportAds: boolean
+}
+
+const Container = styled.div`
+  background-color: rgba(93, 181, 252, 0.2);
+  border-radius: 8px;
+  padding: 8px;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`
+
+const ToggleRow = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 12px 14px;
+  border-radius: 10px;
+  background-color: var(--background1);
+`
+
+const DescriptionRow = styled.div`
+  width: 100%;
+  display: grid; 
+  grid-template-rows: 16px 1fr;
+  grid-template-columns: 16px 1fr;
+  padding: 0 8px;
+  gap: 8px;
+  color: #339AF0;
+  font-family: var(--brave-font-family-non-sherif);
+  letter-spacing: 0.01em;
+  margin-bottom: 7px;
+`
+
+const DescriptionIcon = styled.div`
+  background: url(${InfoIcon}) no-repeat;
+`
+
+const DescriptionTitle = styled.div`
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 18px;
+`
+
+const DescriptionBody = styled.div`
```

Comment:
Q. I wonder if I should prefer more semantic tags over `<div>`. e.g. in this case `<p>` ?

Reply:
The problem with `<p>` is that it sometimes has a bit of default styling which might mess up the perfect padding you have (or you might need to add extra styles to override it). I think what you have is good, but it's up to you :smile:

Reply:
>  you might need to add extra styles to override it

That's exactly why I reverted this to `div`! Then let me leave this as is :)